src/ripple/basics/impl/Log.cpp

```diff
@@ -161,11 +161,11 @@ std::vector<std::pair<std::string, std::string>>
 Logs::partition_severities() const
 {
     std::vector<std::pair<std::string, std::string>> list;
-    std::lock_guard <std::mutex> lock (mutex_);
+    std::lock_guard lock (mutex_);
     list.reserve (sinks_.size());
-    for (auto const& e : sinks_)
-        list.push_back(std::make_pair(e.first,
-            toString(fromSeverity(e.second->threshold()))));
+    for (auto const& [name, sink] : sinks_)
+        list.push_back(std::make_pair(name,
```

Comment:
What are your thoughts on changing `std::make_pair` to `std::pair` thanks to CTAD? Similarly, if we have `std::make_tuple` floating around, we could replace with `std::tuple`.

Reply:
I'll change this to `emplace_back` and get rid of the `std::pair`. Will fix.

Reply:
Fixed in 03289817b `[fold] Use structured binding in another place & rm std::pair ctor:`